src/Flex.tsx

```diff
@@ -3,6 +3,9 @@ import Box, {BoxProps} from './Box'
 
 export type FlexProps = BoxProps
 
+/**
+ * @deprecated Please use the Box component instead.
```

Comment:
Can we provide more specific instructions on how to migrate to the Box component in these deprecation comments?
```suggestion
 * @deprecated Use the Box component instead (i.e. <Flex> → <Box display="flex">)
```